packages/shared/src/components/post/common/SharedPostLink.tsx

```diff
@@ -0,0 +1,42 @@
+import Link from 'next/link';
+import React, { ReactElement, ReactNode } from 'react';
+import { Post } from '../../../graphql/posts';
+import { CombinedClicks } from '../../../lib/click';
```

Comment:
Alternative would have been to import type from.
We have some examples of that in the codebase.

Reply:
> Alternative would have been to import type from.
We have some examples of that in the codebase.

Not exactly sure what you mean here, can you give an example?

Reply:
You can import types specifically, probably something we should leverage more:

```
import type { FilterOnboardingProps } from './FilterOnboarding';
```

Reply:
This is actually importing the type as shown below. Reason why we updated it to become an exported type.

![Screenshot 2023-12-12 at 5 55 01 PM](https://github.com/dailydotdev/apps/assets/13744167/386b4853-91f1-4cd9-8944-074b01f21035)

Reply:
Ah, I understand the `import type` now. Looks to be something we can definitely apply more and more. We just probably need to have a rule when to use it. Like would it still be beneficial if you are already importing something from the same file, etc.

Discussion would be great, indeed.